client/support/support-user/active-dialog.jsx

```diff
@@ -0,0 +1,69 @@
+/**
+ * External Dependencies
+ */
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+
+/**
+ * Internal Dependencies
+ */
+import Dialog from 'components/dialog';
+import FormButton from 'components/forms/form-button';
+import FormLabel from 'components/forms/form-label';
+import EnableLocaleCheckbox from './enable-locale-checkbox';
+import config from 'config';
+import { getCurrentUser } from 'state/current-user/selectors';
+import i18n from 'lib/mixins/i18n'
+
+class SupportUserActiveDialog extends Component {
+
+	render() {
+		const { isVisible, onCloseDialog, onRestoreUser } = this.props;
+
+		const { defaultLocaleSlug, userLocaleSlug } = this.props;
+
+		const buttons = [
+			<FormButton
+				key="close"
+				isPrimary={ false }
+				onClick={ onCloseDialog }>
+					Close
```

Comment:
Might be worth translating this and other strings, unless we've somewhere made an explicit decision not to. At present, users of this feature are English speakers, but maybe that won't always be the case.